_pydevd_bundle/pydevd_extension_api.py

```diff
@@ -0,0 +1,90 @@
+import abc
+
+
+# borrowed from from six
+def _with_metaclass(meta, *bases):
+    """Create a base class with a metaclass."""
+
+    class metaclass(meta):
+        def __new__(cls, name, this_bases, d):
+            return meta(name, bases, d)
+
+    return type.__new__(metaclass, 'temporary_class', (), {})
+
+
+# =======================================================================================================================
+# AbstractResolver
+# =======================================================================================================================
+class _AbstractResolver(_with_metaclass(abc.ABCMeta)):
+    """
+        This class exists only for documentation purposes to explain how to create a resolver.
+
+        Some examples on how to resolve things:
+        - list: get_dictionary could return a dict with index->item and use the index to resolve it later
+        - set: get_dictionary could return a dict with id(object)->object and reiterate in that array to resolve it later
+        - arbitrary instance: get_dictionary could return dict with attr_name->attr and use getattr to resolve it later
+    """
+
+    use_value_repr_instead_of_str = False
```

Comment:
Out of 9 usages in pydevd_resolver only one is `use_value_repr_instead_of_str = True`. Maybe it makes sense to provide a default value since the predominant majority of the uses is expected to want the default?